app/views/mentors/van_drivers.html.erb

```diff
@@ -1,6 +1,29 @@
-<%= javascript_include_tag 'driver_form.js' %>
+<h1>Drivers for <%= @term.is_a?(Quarter) ? @term.title : @term.to_param %></h1>
 
-<h1>Current Trained Drivers</h1>
+<table id="current_drivers">
+	<thead>
+		<tr>
+			<th>Name</th>
+			<th>E-mail</th>
+			<th>Training Date</th>
+			<%- if Customer.link_to_uw? %><th>UWFS Training Date</th><% end -%>
+            <th class="centered">Signed Form?</th>
+            <th class="centered">Convictions?</th>
+			<th colspan=4>Functions</th>
+		</tr>
+	</thead>
+
+	<tbody>
+		<%= render :partial => "mentor_driver", :collection => @current_drivers %>
+		<%- if @mentors.empty? -%>
+			<tr class="empty">
+				<td colspan=10>No <%= Customer.mentors_label %> are currently trained to drive.</td>
+			</tr>
+		<% end -%>
+	</tbody>
+</table>
+
+<h1>All Trained Drivers</h1>
 
 <table>
```

Comment:
You're right! I'll probably ajax load the tables by location, then Give
hsls a parameter to view drivers for their school.
On Mar 27, 2015 10:33 AM, "Matt Harris" <notifications@github.com> wrote:

> In app/views/mentors/van_drivers.html.erb
> <https://github.com/uwdreamproject/DreamSIS/pull/115#discussion_r27314299>
> :
>
> > +            <th class="centered">Convictions?</th>
> > +			<th colspan=4>Functions</th>
> > +		</tr>
> > +	</thead>
> > +
> > +	<tbody>
> > +		<%= render :partial => "mentor_driver", :collection => @current_drivers %>
> > +		<%- if @mentors.empty? -%>
> > +			<tr class="empty">
> > +				<td colspan=10>No <%= Customer.mentors_label %> are currently trained to drive.</td>
> > +			</tr>
> > +		<% end -%>
> > +	</tbody>
> > +</table>
> > +
> > +<h1>All Trained Drivers</h1>
> >
> >  <table>
>
> Either way, you might consider ajax loading the tables...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uwdreamproject/DreamSIS/pull/115/files#r27314299>.
>